pkg/v1/providers/ytt/03_customizations/03_windows/prevent_windows_updates.yaml

```diff
@@ -0,0 +1,15 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+#@ if data.values.IS_WINDOWS_WORKLOAD_CLUSTER:
+#@overlay/match by=overlay.subset({"kind":"KubeadmConfigTemplate"})
```

Comment:
Is there never going to be Linux MachineDeployments in a Windows workload cluster? If so, this is fine, otherwise might need a more specific matcher?

Reply:
Only the control plane. We want this applied to all KubeadmConfigTemplate elements when it's a Windows cluster.